Communication_test.py

```diff
@@ -1,10 +1,19 @@
+import math
 from machine import Pin
 import time
 
 red = Pin(15, Pin.OUT)
 green = Pin(14, Pin.OUT)
 
+
+def string_to_binary(x):
+  l = [(ord(i)) for i in x]
+  m = [(int(bin(i)[2:])) for i in l]
```

Comment:
What does this do?

```suggestion
  m = [bin(i)[2:] for i in l]
  m = [int(item) for item in m]
```

Reply:
Leave a comment explaining why we need to slice [2:]

Reply:
@MIWdlB The comment is now there